manifest.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: powerdummy-populator
+  namespace: relationships
+  labels:
+    app: powerdummy
+    team: relationships
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  revisionHistoryLimit: 5
+  selector:
+    matchLabels:
+      app: powerdummy-populator
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
```

Comment:
Delete the annotations if you are not exposing metrics

Reply:
fixed